File: Python/fibonacci.py
n=int(input("Enter the no of terms of fibonacci series you want:"))
a=0
b=1
print("The fibonacci series is :")
print(a,"\t",b,end="\t")
for i in range(n-2):
    c=a+b
    print(c,end="\t")
    a=b
    b=c

File: JAVA/fibonacci.java
import java.util.Scanner;
public class fibonacci{
    public static void main(String[] args) {
        Scanner sc =new Scanner(System.in);
        System.out.println("Enter the number of fibonacci terms you want:");
        int n=sc.nextInt();
        sc.close();
        int i=0;
        int a=0;
        int b=1;
        int c;
        System.out.println(a);
        System.out.println(b);
        for (i=3;i<=n;i++){
            c=a+b;
            System.out.println(" "+c);
            a=b;
            b=c;
        }
        

    }
}